Workbooks/Workloads/Redis Insights/Redis Insights.workbook

```diff
@@ -477,11 +491,11 @@
               "version": "KqlItem/1.0",
               "query": "InsightsMetrics\r\n| where (Computer in ({OnboardedComputers}) or '*' in ({OnboardedComputers})) \r\n| where Name == \"keyspace_hitrate\"\r\n| make-series Ratio = avg(Val) default=0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Computer",
               "size": 0,
-              "aggregation": 1,
+              "aggregation": 5,
               "showAnalytics": true,
-              "title": "Cache Hit Ratio",
+              "title": "Cache hit ratio",

```

Comment:
Didn't we change this to Cache hit ratio percentage in the alert?

Reply:
yes, needed to figure out the query so was going to defer, but yeah best to just do now - done
Need to update the alert query with this new learnings...